atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -666,7 +666,7 @@ public Void apply(DbDdlTable table) {
             @Override
             public Set<TableReference> apply(SqlConnection conn) {
                 AgnosticResultSet results = conn.selectResultSetUnregisteredQuery(
-                        "SELECT table_name FROM pt_metropolis_table_meta");
+                        "SELECT table_name FROM " + config.shared().metadataTable().getQualifiedName());
```

Comment:
Do we want the metadata table to reflect the prefixing as well?  At the moment it does, yet prior it did.